ddsrouter_test/compose/CMakeLists.txt

```diff
@@ -68,6 +68,10 @@ set(TESTS
     discovery_trigger/none
     discovery_trigger/any/reader
     discovery_trigger/any/writer
+
+    stress/single_router
```

Comment:
When rebasing, you will see that a stress test has been added with a repeater with data fragmentation. We should rethink the names of the tests to go along the new one.

Reply:
Much better. But I think you forgot to store the `simple` test in `simple/reliable` to match the structure of `cloud/reliable`, for example.